### PR TITLE
Blog post: new funding

### PR DESCRIPTION
**Pull request by guusdk**

### CODE REVIEW

**Comment by Fishbowler** [comment]
There's a bunch of line breaks mid-sentence. Are those intentional, or being added by your IDE?

**Comment by guusdk** [comment]
Oh, I didn't think they'd render. Let me remove them

**Comment by guusdk** [comment]
How's that?

**Comment by guusdk** [comment]
(I'm not actually seeing a difference)

File: _posts/2024-10-10-funding.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+date:   2024-10-10 11:10:12 +0200
+author: Guus der Kinderen
+categories:
+---
+
+A new boost in the project's budget will allow us to approximately _double_ the test coverage of our project (and add a couple of nice features)! 
+
+Much of the [XMPP Interop Testing project](https://xmpp-interop-testing.github.io/) was made possible as the work was funded through the NGI0 Core Fund. This is a fund established by [NLnet](https://nlnet.nl/) with financial support from the European Commission's [Next Generation Internet programme](https://ngi.eu/).
+
+It is quite remarkable how far the effects of funding reach: it allowed us to work out our plans to take various, pre-existing bits and bobs, and quickly and efficiently turn a small tool used for internal testing to a proper testing framework for any XMPP server implementation to be able to use. That snowballed in bug fixes for server implementations and improvements to specifications used by many. A relatively small fund thus improved the quality of open standard-based communication used in one shape or another by countless people, daily!
```
Comment (Fishbowler) [inline _posts/2024-10-10-funding.md:14]: A snooty Oxford comma
```suggestion
It is quite remarkable how far the effects of funding reach: it allowed us to work out our plans to take various, pre-existing bits and bobs, and quickly and efficiently turn a small tool used for internal testing to a proper testing framework for any XMPP server implementation to be able to use. That snowballed in bug fixes for server implementations, and improvements to specifications used by many. A relatively small fund thus improved the quality of open standard-based communication used in one shape or another by countless people, daily!
```

File: _posts/2024-10-10-funding.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+We are so happy and grateful for NLnet to boost our project's grant! With the additional work, we will add the following improvements:
```
Comment (Fishbowler) [inline _posts/2024-10-10-funding.md:16]:
```suggestion
We are so happy and grateful to NLnet for boosting our project's grant! With the additional work, we will add the following improvements:
```

**Review by Fishbowler — APPROVED** [review]
Awesome!

Two minor changes, but neither are hills I'll die on.

**Pull request merged**